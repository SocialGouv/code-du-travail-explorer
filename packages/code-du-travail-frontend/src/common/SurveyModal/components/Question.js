import { theme } from "@socialgouv/cdtn-ui";
import styled from "styled-components";

const { fonts, spacings } = theme;

export const Question = styled.div`
  margin-bottom: ${spacings.base};
  font-weight: bold;
  font-size: ${fonts.sizes.headings.small};
`;
