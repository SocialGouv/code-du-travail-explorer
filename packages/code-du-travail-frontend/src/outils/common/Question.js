import React from "react";
import styled from "styled-components";
import { theme } from "@socialgouv/react-ui";

export function Question({ required, children, ...otherProps }) {
  return (
    <Label {...otherProps}>
      {children}
      {required && <Mandatory aria-label="champs obligatoire">*</Mandatory>}
    </Label>
  );
}

const Label = styled.label`
  display: block;
  margin-top: ${theme.spacings.medium};
  margin-bottom: ${theme.spacings.small};
  font-size: ${theme.fonts.sizes.headings.small};
  cursor: ${props => (props.as ? "default" : "pointer")};
`;

const Mandatory = styled.span`
  display: inline-block;
  margin-left: ${theme.spacings.small};
  color: ${({ theme }) => theme.error};
  font-weight: 700;
`;
