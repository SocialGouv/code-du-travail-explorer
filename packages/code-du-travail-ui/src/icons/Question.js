import React from "react";
import PropTypes from "prop-types";

const Question = props => (
  <svg viewBox="0 0 50 50" {...props}>
    <title>question</title>
    <g fill={props.fill || "#000"} fillRule="evenodd">
      <path d="M33.045 30.728a.951.951 0 0 0-.955.946v6.533c0 2.61-2.143 4.733-4.775 4.733H17.003a.951.951 0 0 0-.955.947l-.001 1.882-2.28-2.26a.955.955 0 0 0-.876-.569H6.685c-2.633 0-4.775-2.123-4.775-4.733v-14.58c0-2.61 2.142-4.734 4.775-4.734h9.647a.951.951 0 0 0 .955-.946.951.951 0 0 0-.955-.947H6.685C3 17 0 19.973 0 23.627v14.58c0 3.654 2.999 6.627 6.685 6.627h5.717l3.922 3.889a.96.96 0 0 0 1.04.205.946.946 0 0 0 .59-.874l.003-3.22h9.358C31 44.834 34 41.86 34 38.207v-6.533a.951.951 0 0 0-.955-.946z" />
      <path d="M35.201 0H32.8C25.189 0 19 6.195 19 13.81s6.19 13.811 13.798 13.811h2.403a13.82 13.82 0 0 0 3.502-.45l3.55 3.551a.949.949 0 0 0 1.62-.672v-5.498a13.942 13.942 0 0 0 3.627-4.475c.995-1.952 1.5-4.06 1.5-6.267C49 6.195 42.81 0 35.201 0zm7.152 23.33a.95.95 0 0 0-.379.76v3.667l-2.323-2.324a.948.948 0 0 0-.95-.236c-1.13.347-2.307.524-3.5.524H32.8c-6.562 0-11.9-5.343-11.9-11.91 0-6.568 5.338-11.91 11.9-11.91H35.2c6.562 0 11.9 5.342 11.9 11.91 0 3.775-1.73 7.245-4.748 9.52z" />
      <path d="M37.99 10.71a4.005 4.005 0 0 0-3.708-3.7A4 4 0 0 0 30 10.99a.944.944 0 0 0 1.89 0c0-.59.238-1.136.67-1.539a2.099 2.099 0 0 1 1.591-.56 2.109 2.109 0 0 1 1.954 1.948 2.096 2.096 0 0 1-1.641 2.206 1.785 1.785 0 0 0-1.409 1.748v2.264a.944.944 0 0 0 1.89 0V14.87a3.971 3.971 0 0 0 3.045-4.16zM34.707 19.293A1.01 1.01 0 0 0 34 19a1.01 1.01 0 0 0-.707.293A1.008 1.008 0 0 0 33 20c0 .264.107.522.293.708a1.006 1.006 0 0 0 1.414 0A1.01 1.01 0 0 0 35 20c0-.263-.107-.52-.293-.707zM26.048 30H5.952C5.426 30 5 30.448 5 31s.426 1 .952 1h20.096c.526 0 .952-.448.952-1s-.426-1-.952-1zM26.707 35.293A1.007 1.007 0 0 0 26 35c-.263 0-.521.107-.707.293A1.007 1.007 0 0 0 25 36c0 .263.107.521.293.707.186.186.444.293.707.293.263 0 .521-.107.707-.293.186-.186.293-.443.293-.707 0-.263-.107-.521-.293-.707zM23.009 35H5.99A.996.996 0 0 0 5 36c0 .552.444 1 .991 1H23.01A.996.996 0 0 0 24 36c0-.552-.444-1-.991-1zM20.036 25H5.964C5.432 25 5 25.448 5 26s.432 1 .964 1h14.072c.532 0 .964-.448.964-1s-.432-1-.964-1z" />
    </g>
  </svg>
);

export default Question;

Question.propTypes = {
  fill: PropTypes.string
};
