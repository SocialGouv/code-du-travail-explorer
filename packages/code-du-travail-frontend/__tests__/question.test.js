import React from "react";
import { render } from "@testing-library/react";
import Question from "../pages/question.js";

describe("<Question />", () => {
  it("should render", () => {
    const { container } = render(<Question />);
    expect(container).toMatchSnapshot();
  });
});
