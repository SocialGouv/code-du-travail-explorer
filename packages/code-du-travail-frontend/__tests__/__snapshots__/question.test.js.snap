// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Question /> should render 1`] = `
.c14:not(:focus):not(:active) {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  -webkit-clip: rect(0,0,0,0);
  clip: rect(0,0,0,0);
  border: 0;
}

.c44 {
  margin: 0;
  padding: 0;
  list-style-type: none;
}

.c1 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
}

.c1 > *:first-child {
  margin-top: 0;
}

.c1 > *:last-child {
  margin-bottom: 0;
}

.c21 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  max-width: 46.25rem;
}

.c21 > *:first-child {
  margin-top: 0;
}

.c21 > *:last-child {
  margin-bottom: 0;
}

.c38 {
  max-width: 1200px;
  margin: 0 auto;
  padding: 0 1.25rem;
  padding: 0;
  max-width: 46.25rem;
}

.c38 > *:first-child {
  margin-top: 0;
}

.c38 > *:last-child {
  margin-bottom: 0;
}

.c7 {
  padding: 1.25rem 0;
  background-color: #fff;
}

.c20 {
  padding: 1.25rem 0;
}

.c22 {
  padding: 0.625rem 1.25rem;
  color: #434956;
  border-width: 1px;
  border-style: solid;
  border-radius: 0.25rem;
  border-color: #c9d3df;
  background-color: #ebeff3;
}

.c22 > *:first-child {
  margin-top: 0;
}

.c22 > *:last-child {
  margin-bottom: 0;
}

.c19 {
  margin-bottom: 1rem;
  padding: 0.625rem 1.25rem;
  border: 1px solid transparent;
  border-radius: 0.25rem;
  color: #0c0c0e;
  background: #fee5ad;
  border-color: #fdd57b;
}

.c19 > *:first-child {
  margin-top: 0;
}

.c19 > *:last-child {
  margin-bottom: 0;
}

.c39 {
  padding: 0.625rem 1rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  text-align: center;
  line-height: inherit;
  font-size: 1rem;
  font-weight: 600;
  vertical-align: middle;
  border-style: solid;
  border-width: 1px 1px 2px 1px;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  color: #fff;
  background: #005994;
  border-color: #005994;
  border-bottom-color: #003a61;
}

.c39:not([disabled]):hover,
.c39:not([disabled]):focus {
  background: #0068ae;
  color: #fff;
}

.c39:not([disabled]):active {
  color: #fff;
  background: #0078c7;
  border-width: 2px 1px 1px 1px;
  border-color: #005994;
  outline: none;
}

.c39[aria-pressed="true"] {
  color: #fff;
  background: #0068ae;
  border-width: 2px 1px 1px 1px;
  border-color: #005994;
  border-top-color: #003a61;
  box-shadow: inset 0 1px 2px 0 #003a61;
}

.c39[aria-pressed="true"]:not([disabled]):hover,
.c39[aria-pressed="true"]:not([disabled]):focus {
  background-color: #005994;
}

.c39[aria-pressed="true"]:not([disabled]):active {
  border-top-color: #003a61;
}

.c39[disabled] {
  cursor: not-allowed;
  color: rgba(255,255,255,0.4);
}

.c34 {
  padding: 0.625rem 1rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  text-align: center;
  line-height: inherit;
  font-size: 1rem;
  font-weight: 600;
  vertical-align: middle;
  border-style: solid;
  border-width: 1px 1px 2px 1px;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  padding: 1rem;
  color: #434956;
  line-height: 0;
  border: none;
}

.c34:hover {
  color: #8c94a6;
}

.c34:active {
  position: relative;
  top: 1px;
}

.c29 {
  box-sizing: border-box;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  min-height: 48px;
  width: 100%;
  background-color: white;
  border-style: solid;
  border-width: 1px;
  border-radius: 0.25rem;
  -webkit-animation: iowhYu 0.3s ease-out;
  animation: iowhYu 0.3s ease-out;
  border-color: #fee5ad;
  box-shadow: 0 5px 10px 0 #c9d3df;
}

.c30 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 1rem;
  background-color: #fee5ad;
}

.c31 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  padding: 1rem;
  color: #434956;
  text-align: left;
}

.c33 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c28 {
  position: fixed;
  bottom: 0;
  padding: 1rem;
  width: 100%;
  z-index: 1000;
}

.c32 {
  padding: 0 0.25rem;
}

.c13 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  text-align: left;
}

.c16 ul {
  z-index: 100;
}

.c16 li[role="option"]:nth-child(2n + 1) {
  background: #f5f7fa;
}

.c6 {
  box-shadow: 0 10px 10px -10px #c9d3df;
}

.c9 {
  margin-top: 0;
  font-size: 1.875rem;
  line-height: 1.4;
  color: #006ab2;
}

.c10 {
  font-size: 1rem;
}

.c8 {
  position: relative;
  padding: 1rem 0;
  text-align: center;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 auto;
  padding: 0;
  position: relative;
  width: 70%;
}

.c12 {
  position: absolute;
  left: 0;
  top: 0;
  margin: 1rem 0 0 1.25rem;
  width: 1.3rem;
  height: 1.3rem;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0;
  padding: 0 1rem 0 4rem;
  width: 100%;
  height: 50px;
  font-size: inherit;
  font-family: inherit;
  color: inherit;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  background: #f9f9fc;
  border: 1px solid #c9d3df;
  border-radius: 0.25rem;
  -webkit-transition: border 250ms ease;
  transition: border 250ms ease;
}

.c15:focus {
  border-color: #006be6;
  outline: none;
}

.c17 {
  padding: 0.625rem 1rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  text-align: center;
  line-height: inherit;
  font-size: 1rem;
  font-weight: 600;
  vertical-align: middle;
  border-style: solid;
  border-width: 1px 1px 2px 1px;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  color: #fff;
  background: #006be6;
  border-color: #006be6;
  border-bottom-color: #0053b3;
  margin-left: 0.5rem;
}

.c17:not([disabled]):hover,
.c17:not([disabled]):focus {
  background: #07f;
  color: #fff;
}

.c17:not([disabled]):active {
  color: #fff;
  background: #1a85ff;
  border-width: 2px 1px 1px 1px;
  border-color: #006be6;
  outline: none;
}

.c17[aria-pressed="true"] {
  color: #fff;
  background: #07f;
  border-width: 2px 1px 1px 1px;
  border-color: #006be6;
  border-top-color: #0053b3;
  box-shadow: inset 0 1px 2px 0 #0053b3;
}

.c17[aria-pressed="true"]:not([disabled]):hover,
.c17[aria-pressed="true"]:not([disabled]):focus {
  background-color: #006be6;
}

.c17[aria-pressed="true"]:not([disabled]):active {
  border-top-color: #0053b3;
}

.c17[disabled] {
  cursor: not-allowed;
  color: rgba(255,255,255,0.4);
}

.c23 {
  margin-bottom: 1rem;
}

.c24 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c26 {
  padding: 0.625rem 1rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  text-align: center;
  line-height: inherit;
  font-size: 1rem;
  font-weight: 600;
  vertical-align: middle;
  border-style: solid;
  border-width: 1px 1px 2px 1px;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  color: #0c0c0e;
  background: #eaeaea;
  border-color: #eaeaea;
  border-bottom-color: #d1d1d1;
  min-width: 7rem;
}

.c26:not([disabled]):hover,
.c26:not([disabled]):focus {
  background: #f7f7f7;
  color: #18181c;
}

.c26:not([disabled]):active {
  color: #242429;
  background: #fff;
  border-width: 2px 1px 1px 1px;
  border-color: #eaeaea;
  outline: none;
}

.c26[aria-pressed="true"] {
  color: #18181c;
  background: #f7f7f7;
  border-width: 2px 1px 1px 1px;
  border-color: #eaeaea;
  border-top-color: #d1d1d1;
  box-shadow: inset 0 1px 2px 0 #d1d1d1;
}

.c26[aria-pressed="true"]:not([disabled]):hover,
.c26[aria-pressed="true"]:not([disabled]):focus {
  background-color: #eaeaea;
}

.c26[aria-pressed="true"]:not([disabled]):active {
  border-top-color: #d1d1d1;
}

.c26[disabled] {
  cursor: not-allowed;
  color: rgba(12,12,14,0.4);
}

.c26 + .c25 {
  margin-left: 1rem;
}

.c27 {
  padding: 0.625rem 1rem;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  text-align: center;
  line-height: inherit;
  font-size: 1rem;
  font-weight: 600;
  vertical-align: middle;
  border-style: solid;
  border-width: 1px 1px 2px 1px;
  border-radius: 0.25rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  color: #fff;
  background: #006be6;
  border-color: #006be6;
  border-bottom-color: #0053b3;
  min-width: 7rem;
}

.c27:not([disabled]):hover,
.c27:not([disabled]):focus {
  background: #07f;
  color: #fff;
}

.c27:not([disabled]):active {
  color: #fff;
  background: #1a85ff;
  border-width: 2px 1px 1px 1px;
  border-color: #006be6;
  outline: none;
}

.c27[aria-pressed="true"] {
  color: #fff;
  background: #07f;
  border-width: 2px 1px 1px 1px;
  border-color: #006be6;
  border-top-color: #0053b3;
  box-shadow: inset 0 1px 2px 0 #0053b3;
}

.c27[aria-pressed="true"]:not([disabled]):hover,
.c27[aria-pressed="true"]:not([disabled]):focus {
  background-color: #006be6;
}

.c27[aria-pressed="true"]:not([disabled]):active {
  border-top-color: #0053b3;
}

.c27[disabled] {
  cursor: not-allowed;
  color: rgba(255,255,255,0.4);
}

.c27 + .c25 {
  margin-left: 1rem;
}

.c18 {
  margin: 20%;
  font-size: 1.875rem;
  text-align: center;
}

.c0 {
  color: white;
  background: currentColor;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1.25rem 0;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
}

.c3:link {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c4 {
  width: 5rem;
  height: 100%;
  margin-right: 1rem;
}

.c5 {
  font-size: 1.3rem;
  line-height: 1.1;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c36 {
  text-align: center;
}

.c37 {
  margin-bottom: 1.25rem;
  text-align: left;
}

.c40 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c42 + .c41 {
  padding-left: 1rem;
}

.c43 {
  font-size: 1rem;
  font-weight: bold;
}

.c45 {
  margin: 0.5rem 0;
}

.c45 a,
.c45 a:visited {
  color: #434956;
}

@media print {
  .c1 {
    max-width: 100%;
    padding: 0;
  }
}

@media print {
  .c21 {
    max-width: 100%;
    padding: 0;
  }
}

@media print {
  .c38 {
    max-width: 100%;
    padding: 0;
  }
}

@media (max-width:600px) {
  .c22 {
    padding: 0.625rem 1.25rem;
  }
}

@media print {
  .c22 {
    border: none;
    padding: 0 5pt;
  }
}

@media print {
  .c6 {
    display: none;
  }
}

@media (max-width:980px) {
  .c11 {
    width: 100%;
  }
}

@media (max-width:600px) {
  .c11 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    height: auto;
    width: 100%;
  }
}

@media (max-width:600px) {
  .c12 {
    display: none;
  }
}

@media (max-width:600px) {
  .c15 {
    height: 50px;
    padding: 0 1rem;
  }
}

@media (max-width:600px) {
  .c17 {
    margin-left: 0;
    margin-top: 0.625rem;
  }
}

@media print {
  .c35 {
    display: none;
  }
}

@media (max-width:600px) {
  .c40 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:600px) {
  .c42 + .c41 {
    padding-left: 0;
  }
}

<div>
  <header
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <a
          class="c3"
          href="/?q="
          title="Code du travail numérique - retour à l'accueil"
        >
          <img
            alt="symbole de la Marianne, site officiel du gouvernement"
            class="c4"
            src="/static/assets/img/marianne.svg"
          />
          <span
            class="c5"
          >
            Code du travail
            <br />
            numérique
          </span>
        </a>
      </div>
    </div>
  </header>
  <main
    id="main"
  >
    <div
      class="c6 c7"
    >
      <div
        class="c1"
      >
        <div
          class="c8"
        >
          <p
            class="c9"
          >
            Posez votre question sur le droit du travail
            <br />
            <a
              class="c10"
              href="/droit-du-travail"
            >
              Le droit du travail, c‘est quoi ?
            </a>
          </p>
          <form
            class="c11"
          >
            <svg
              class="c12"
              viewBox="0 0 20 20"
            >
              <title />
              <path
                d="M17.876 8.939c0 2.197-.819 4.208-2.123 5.773l4.023 4.022a.72.72 0 0 1 0 1.043.735.735 0 0 1-.522.223.736.736 0 0 1-.52-.223l-4.023-4.023a8.813 8.813 0 0 1-5.772 2.123C4.022 17.877 0 13.855 0 8.939 0 4.022 4.023 0 8.939 0c4.915 0 8.937 4.023 8.937 8.939zM9.001 16.5c4.124 0 7.499-3.375 7.499-7.5S13.125 1.5 9 1.5C4.877 1.5 1.5 4.875 1.5 9s3.376 7.5 7.5 7.5z"
                fill="currentColor"
                fill-rule="evenodd"
              />
            </svg>
            <label
              class="c13"
              for="main-search-input"
            >
              <span
                class="c14"
              >
                Rechercher
              </span>
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-owns="react-autowhatever-cdtn-documents-suggester"
                role="combobox"
                style="flex: 1;"
              >
                <input
                  aria-autocomplete="list"
                  aria-controls="react-autowhatever-cdtn-documents-suggester"
                  autocomplete="off"
                  class="c15"
                  id="main-search-input"
                  name="query"
                  placeholder="Recherche"
                  type="search"
                  value=""
                />
                <div
                  class="c16"
                  id="react-autowhatever-cdtn-documents-suggester"
                  role="listbox"
                />
              </div>
            </label>
            <button
              aria-pressed="false"
              class="c17"
              type="submit"
            >
              Rechercher
            </button>
          </form>
        </div>
      </div>
    </div>
    <div
      class="c18 c1"
    >
      <div
        class="c19"
      >
        Cette question n'a pas été trouvée
      </div>
    </div>
    <div
      class="c20"
    >
      <div
        class="c21"
      >
        <div
          class="c22"
        >
          <h3
            class="c23"
          >
            Avez-vous trouvé la réponse à votre question ?
          </h3>
          <p
            class="c24"
          >
            <button
              aria-pressed="false"
              class="c25 c26"
            >
              Non
            </button>
            <button
              aria-pressed="false"
              class="c25 c27"
            >
              Oui
            </button>
          </p>
        </div>
      </div>
    </div>
    <div
      class="c28"
    >
      <div
        class="c29"
      >
        <div
          class="c30"
        >
          <svg
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M10.29 3.86L1.82 18a2 2 0 0 0 1.71 3h16.94a2 2 0 0 0 1.71-3L13.71 3.86a2 2 0 0 0-3.42 0z"
            />
            <line
              x1="12"
              x2="12"
              y1="9"
              y2="13"
            />
            <line
              x1="12"
              x2="12"
              y1="17"
              y2="17"
            />
          </svg>
        </div>
        <div
          class="c31"
          role="alert"
        >
          Ce site est en cours de construction, la fiabilité des réponses qui s’y trouvent ne sont pas garanties.
          <a
            class="c32"
            href="https://www.legifrance.gouv.fr/affichTexteArticle.do;jsessionid=AE9DCF75DDCF0465784CEE0E7D62729F.tplgfr37s_2?idArticle=JORFARTI000035607420&cidTexte=JORFTEXT000035607388&dateTexte=29990101&categorieLien=id"
            rel="noopener noreferrer"
            target="_blank"
            title="consulter l'ordonance"
          >
            L’ouverture officielle du site
          </a>
          est prévue pour 2020.
        </div>
        <div
          class="c33"
        >
          <button
            aria-label="Fermer"
            aria-pressed="false"
            class="c34"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="18"
                x2="6"
                y1="6"
                y2="18"
              />
              <line
                x1="6"
                x2="18"
                y1="6"
                y2="18"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
  </main>
  <footer
    class="c35"
  >
    <div
      class="c20"
    >
      <div
        class="c36 c1"
      >
        <h2>
          Besoin d’un accompagnement personnalisé ?
        </h2>
        <div
          class="c37 c38"
        >
          Les services de renseignement en droit du travail peuvent vous donner des informations juridiques générales relatives au Code du travail, aux conventions collectives, à la jurisprudence. Ils peuvent également vous conseiller et vous orienter dans vos démarches.
        </div>
        <button
          aria-pressed="false"
          class="c39"
        >
          Contacter les services de renseignement
        </button>
      </div>
    </div>
    <div
      class="c20"
    >
      <div
        class="c40 c1"
      >
        <div
          class="c41 c42"
        >
          <h3
            class="c43"
          >
            Code du travail numérique
          </h3>
          <ul
            class="c44"
          >
            <li
              class="c45"
            >
              <a
                href="/droit-du-travail"
              >
                Le droit du travail
              </a>
            </li>
            <li
              class="c45"
            >
              <a
                href="/a-propos"
              >
                À propos
              </a>
            </li>
            <li
              class="c45"
            >
              <a
                href="/mentions-legales"
              >
                Mentions légales
              </a>
            </li>
            <li
              class="c45"
            >
              <a
                href="mailto:codedutravail@beta.gouv.fr"
              >
                Contact
              </a>
            </li>
          </ul>
        </div>
        <div
          class="c41 c42"
        >
          <h3
            class="c43"
          >
            Aidez-nous à améliorer cet outil
          </h3>
          <ul
            class="c44"
          >
            <li
              class="c45"
            >
              <a
                href="https://github.com/SocialGouv/code-du-travail-numerique/tree/vX.Y.Z"
                rel="noopener noreferrer"
                target="_blank"
              >
                Contribuer sur Github
              </a>
            </li>
            <li
              class="c45"
            >
              <a
                href="https://github.com/SocialGouv/code-du-travail-numerique/releases/tag/vX.Y.Z"
                rel="noopener noreferrer"
                target="_blank"
              >
                Journal des modifications
              </a>
            </li>
          </ul>
        </div>
        <div
          class="c41 c42"
        >
          <h3
            class="c43"
          >
            En collaboration avec
          </h3>
          <ul
            class="c44"
          >
            <li
              class="c45"
            >
              <a
                href="https://travail-emploi.gouv.fr/ministere/organisation/article/dgt-direction-generale-du-travail"
                rel="noopener noreferrer"
                target="_blank"
              >
                La Direction Générale du Travail
              </a>
            </li>
            <li
              class="c45"
            >
              <a
                href="https://incubateur.social.gouv.fr/"
                rel="noopener noreferrer"
                target="_blank"
              >
                L’incubateur des ministères sociaux
              </a>
            </li>
            <li
              class="c45"
            >
              <a
                href="https://beta.gouv.fr/"
                rel="noopener noreferrer"
                target="_blank"
              >
                beta.gouv.fr
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
